tools/build_config/Jenkinsfile

```diff
@@ -73,25 +73,47 @@ pipeline {
       }
     }
 
-    stage('Clone-Herbert') {
+    stage('Copy-Herbert') {
       steps {
-        copyArtifacts(
-          filter: 'build/Herbert-*',
-          fingerprintArtifacts: true,
-          projectName: "PACE-neutrons/Herbert/${base_job_name}",
-          selector: lastSuccessful()
-        )
         script {
-          if (isUnix()) {
-            sh '''
-              archive_name="\$(find -name Herbert-*.tar.gz)"
-              tar --wildcards --strip-components=1 -xf \$archive_name */Herbert
-            '''
-          }
-          else {
-            powershell './tools/build_config/extract_herbert_artifact.ps1'
+          if (env.RELEASE_TYPE == "nightly") {
+            copyArtifacts(
+              filter: 'build/Herbert-*',
+              fingerprintArtifacts: true,
+              projectName: "PACE-neutrons/Herbert/${base_job_name}",
+              selector: lastSuccessful()
+            )
+
+            if (isUnix()) {
+              sh '''
+                archive_name="\$(find -name Herbert-*.tar.gz)"
+                tar --wildcards --strip-components=1 -xf \$archive_name */Herbert
+              '''
+            }
+            else {
+              powershell './tools/build_config/extract_herbert_artifact.ps1'
+            }
+
+          } else {
+            if (isUnix()) {
+              sh '''
+                ./tools/build_config/clone_herbert_branch.sh \
+                    "--cmake_flags \"-DHorace_RELEASE_TYPE=\$RELEASE_TYPE\" \
+                    --matlab_release \$MATLAB_VERSION \
+                    --build_tests OFF
+              '''
+            } else {
+              powershell '''
+                ./tools/build_config/clone_herbert_branch.ps1 \
+                    -cmake_flags ""-DHerbert_RELEASE_TYPE=\$env:RELEASE_TYPE"" \
+                    -matlab_release \$env:MATLAB_VERSION \
+                    -build_tests OFF
```

Comment:
there's no need to pass the `-build_tests OFF` argument -- we will never run Herbert tests from the Horace pipeline so that can be added directly to the build commands in the scripts below before the vararg piece.